.github/workflows/run-fresh.yml

```diff
@@ -0,0 +1,80 @@
+name: Fresh
```

Comment:
Maybe?
```suggestion
name: Integration Tests
```
And also change the filename to `run-integration-tests.yml`? would fit nicely into the existing ones:
![image](https://user-images.githubusercontent.com/87493/92789474-2d1c2d80-f3ab-11ea-80ef-9f01690325ea.png)